### PR TITLE
Remove startup.sh and add govuk-docker to README

### PR DESCRIPTION
**Pull request by barrucadu**
There was a reference to github.gds! 😱 

---

[Trello card](https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh)

### CODE REVIEW

**Review by benthorner — COMMENTED** [review]
Thanks for tackling this one - sorry it's not actually that straightforward 😬. I've left a few comments where I think it's worth double-checking the rake tasks we're proposing to remove. The dependencies are an awkward one: this app has quite a few, but there's no real context about the features they enable (otherwise I've normally mentioned them in the intro at the start of the README). I think it's OK to just remove them all 👍.

A couple of little extra things we could do while we're here:

- Remove the screenshots. We've done this elsewhere as they add little value for the large space and scrolling they entail.
- Rename `doc -> docs`. I thought we did this for most repos, but perhaps this is one that was missed by @ChrisBAshton.
- Remove the "Link Checker API" section. This doesn't describe anything about the code in this app, and the link at the end is broken.
- Demote "Working with Service Sign In pages" to a generic "Further documentation" section towards the end.

File: README.md [hunk 1]
```diff
@@ -44,52 +44,20 @@ See the [README](lib/service_sign_in/README.md) for more details.
...
-bundle exec rake local_transactions:import SOURCE=/path/to/local_authority_service_details.CSV
-```
-
-- [statsd](https://github.com/etsy/statsd/) - this application uses [statsd-ruby](http://rubygems.org/gems/statsd-ruby) to send metrics to statsd. If a statsd process isn't present on the server it won't matter as statsd-ruby sends metrics over UDP. If a statsd process is present then
-it'll send strings with the respective increment/decrement/gauge function to use.
-- [asset-manager](https://github.com/alphagov/asset-manager) - manages uploaded assets (images, PDFs, videos etc.). Publisher needs an OAuth bearer token in order to authenticate with Asset Manager. By default, this is loaded from the `PUBLISHER_ASSET_MANAGER_BEARER_TOKEN` environment variable in `config/initializers/gds_api.rb`, which is automatically provided if using the development VM.
-Otherwise, to obtain this bearer token you should create an API user in the signonotron2 application. In the signonotron2 directory, to generate the bearer token you need, run:
-
-```shell
-rake api_clients:create[publisher,publisher@example.com,asset-manager,signin]
```
[the remark this answers is not in the document]
Comment (barrucadu) [reply README.md:66]: Yes, this is a task in Signon, and also there's a web UI to create tokens.

File: README.md [hunk 2]
```diff
@@ -44,52 +44,20 @@ See the [README](lib/service_sign_in/README.md) for more details.
...
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies.  Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-- [imminence](https://github.com/alphagov/imminence) - provides geographical search tools
-- [content-store](https://github.com/alphagov/content-store) - new central storage of published content on GOV.UK
-- [publishing-api](https://github.com/alphagov/publishing-api) - will provide workflow for all content published to GOV.UK - creating a new document, publishing it, etc. Content published here will end up in the content-store
-- [govuk-content-schemas](http://github.com/alphagov/govuk-content-schemas) - defines the schemas for new-style document formats. Required to run the tests.
+**Use GOV.UK Docker to run any commands that follow.**
 
-```shell
-bundle exec rake local_transactions:import SOURCE=/path/to/local_authority_service_details.CSV
```
[the remark this answers is not in the document]
Comment (barrucadu) [reply README.md:57]: The task doesn't exist any more, it looks like it [got replaced with a task to download from Directgov in 2012](https://github.com/alphagov/publisher/commit/c2d944b05a9c2021eff6fad0ab5bc40a4a6a7fa3#diff-77e05a9328ca12a2c93c12594fd55d77edee7d018ec9b3b61e752a32dc0d14cb) which was then [deleted in 2016](https://github.com/alphagov/publisher/commit/725496cdf6ba094c61ed81f353d48fc1d8fd6be8#diff-77e05a9328ca12a2c93c12594fd55d77edee7d018ec9b3b61e752a32dc0d14cb).

File: README.md [hunk 3]
```diff
@@ -4,11 +4,8 @@ Publisher is the primary content design app for GOV.UK. It provides the user int
 entering all the key editorial formats and an API so other apps (primarily frontend) can
```
Comment (benthorner) [inline README.md:4]:
```suggestion
```

Optional: all of this seems wrong: it's neither the key app, nor has any particular API that I'm aware of. The only thing that's useful in this paragraph appears to be the pseudonym sentence.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (benthorner) [reply README.md:66]: Ahh, yes, I assumed it was another random task in this app, and not Signon. Agree it's not necessary for setup 👍.

**Review by benthorner — APPROVED** [review]
Nice one, thanks for addressing all those comments. I've just left an optional one, in case you feel so inclined (or it can be someone else's puzzle for the future: what does this app actually do? 😆)

**Pull request merged**